docs/plugins/subtitles.md

```diff
@@ -0,0 +1,63 @@
+# Subtitles
+
+subtitle plugin allows you to fetch and parse the subtitle file from the given URL and you can read the subtitle text from the parsed file based on the current videoplayback time.
+
+## Usage
+
+If you want to access Subtitles in your App code directly, import the *Subtitles* plugin from the Lightning SDK:
+
+```js
+import { Subtitles } from '@lightningjs/sdk'
```

Comment:
Now that I think of it, this requires a change in the Lightning-SDK as well, so that this is exposed through that package as well. Maybe users could import it straight from the `@metrological/sdk`?

Reply:
Since in the latest versions `@metrological/sdk` is dependent in `@lightningjs/sdk`  we can directly import `Subtitles` from @metrological/sdk, but it could be better if it's exposed from @lightningjs/sdk.